src/RadixDlt.NetworkGateway.PostgresIntegration/DataAggregatorBuilderExtensions.cs

```diff
@@ -93,7 +93,7 @@ public static DataAggregatorBuilder AddPostgresPersistenceCore(this DataAggregat
             .AddSingleton<IPendingTransactionResubmissionService, PendingTransactionResubmissionService>()
             .AddSingleton<IPendingTransactionPrunerService, PendingTransactionPrunerService>()
             .AddSingleton<MetricsInterceptor>()
-            .AddScoped<IDapperWrapper, DapperWrapper>();
+            .AddSingleton<IDapperWrapper, DapperWrapper>();
```

Comment:
This type cannot be a singleton as it relies on `IOptionsSnapshot` for `slowQueriesLoggingOptions`. Isn't it throwing an exception?